tests/foreman/ui/test_ansible.py

```diff
@@ -336,3 +316,27 @@ def test_positive_remove_role_in_new_ui(self):
     :expectedresults: The Role is successfully removed from the Host, and no longer shows
         up on the UI
     """
+    SELECTED_ROLE = 'RedHatInsights.insights-client'
+
+    location = function_host.location.read()
+    organization = function_host.organization.read()
```

Comment:
Are these lines read loc and org of the RHEL Host which is being created using fixture `function_host` ?